README.md

```diff
@@ -120,9 +120,9 @@ Data which is stored upon startup includes:
 
 * It is easiest to use the development virtual machine to work on this project. Follow [these instructions](RemoteDesktopToVM.md) to set up and connect to the virtual machine using your Gordon account.
 
-* If this is your first time on the virtual machine, you will need to clone this repository. You can do this by using Git Bash.
+* If this is your first time on the virtual machine, you will need to clone this repository. You can do this by using Git Bash. It is possible that you will need to add the SSH key to your Git account, instructions to generate one can be found [here](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh). You will then need to `cd ssh` then `cat id_rsa.pub`, copy the output and go to your github settings and paste it in your SSH keys.
```

Comment:
I'm not sure it's universally agreed upon, but I'd encourage you to avoid using links with the link text "here".  You can nearly always reword things to read more naturally and so that link text is descriptive of what it points to.  Something like "You will also want to [create an SSH public key (if you don't already have one) and add it to your Github account](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh)."